Update openmpi-jobs.md (#143)
osg-htc/user-documentation: documentation/htc_workloads/specific_resources/openmpi-jobs.md

becomes more difficult due to the limited number of resources).

> To get started, first compile your code using the OpenMPI container. You can create your own OpenMPI container or use the one that is available on dockerhub. OSG has a `openmpi` container. To compile your code using the OSG provided image, load the imagae first. 

Reviewer: ```suggestion
To get started, first compile your code using the OpenMPI container. You can create your own OpenMPI container or use the one that is available on dockerhub. OSG has a `openmpi` container. To compile your code using the OSG provided image, start running the container first. 
```

> To get started, first compile your code using the OpenMPI container. You can create your own OpenMPI container or use the one that is available on dockerhub. OSG has a `openmpi` container. To compile your code using the OSG provided image, load the imagae first. 

Reviewer 2: ```suggestion
To get started, first compile your code using the OpenMPI container. You can create your own OpenMPI container or use one that is available on Docker Hub. The OSPool also hosts an `openmpi` container you can use. To compile your code using the OSG provided image, load the image first. 
```

    $ apptainer shell /ospool/uc-shared/public/OSG-Staff/openmpi.sif
    $ mpicc -o hello hello.c 

The `hello.c` is an example hello world code that can be executed using multiple processors. The code is given below:

```
#include <mpi.h>
#include <stdio.h>

int main(int argc, char** argv) {
        MPI_Init(NULL, NULL);
        int world_size;
        MPI_Comm_size(MPI_COMM_WORLD, &world_size);
        int world_rank;
        MPI_Comm_rank(MPI_COMM_WORLD, &world_rank);
        char processor_name[MPI_MAX_PROCESSOR_NAME];
        int name_len;
        MPI_Get_processor_name(processor_name, &name_len);
        printf("Hello world from processor %s, rank %d out of %d processors\n", processor_name, world_rank, world_size);
        MPI_Finalize();
}
```
After compiling the code, you can test the executable locally using `mpiexec`:

    $ mpiexec -n 4 hello
    Hello world from processor ap21.uc.osg-htc.org, rank 0 out of 4 processors
    Hello world from processor ap21.uc.osg-htc.org, rank 1 out of 4 processors
    Hello world from processor ap21.uc.osg-htc.org, rank 2 out of 4 processors
    Hello world from processor ap21.uc.osg-htc.org, rank 3 out of 4 processors

> 

Reviewer 2: Add a comment about exiting from the container after you compile/test.

Author: Done
